processing/include/Process.h

```diff
@@ -71,13 +71,28 @@ class Process {
         }
 
         /**
+<<<<<<< HEAD
```

Comment:
you have markings here from a manual overlap merge. When did you add skip events like this?

Reply:
I added it near the beginning. I wanted to use skip events to perform debugging.